Projects/Features/SignupFeature/Sources/SchoolCodeViewModel.swift

```diff
@@ -0,0 +1,37 @@
+import BaseFeature
+import Combine
+import DomainModule
+
+final class SchoolCodeViewModel: BaseViewModel {
+
```

Comment:
```suggestion
final class SchoolCodeViewModel: BaseViewModel {
```
어.. 첫줄 공백 없이 가는거 어떠신가요